libs/src/services/ethContracts/claimsManagerClient.js

```diff
@@ -88,13 +88,16 @@ class ClaimsManagerClient extends ContractClient {
   }
 
   // Returns boolean indicating whether a claim is considered pending
-  async initiateRound () {
+  async initiateRound (txRetries = 5) {
```

Comment:
why this change?

Reply:
This is so that when we have the script to initiate rounds, on failure they wouldn't spend gas on retrying.

Reply:
sounds great! send it!